tests/Unit/Domain/ShoppingListTest.php

```diff
@@ -71,8 +71,8 @@ public function testAddItem(): void
      */
     public function testSetArchived(ShoppingList $list): void
     {
-        $this->assertSame($list, $list->setArchived(false));
-        $this->assertSame(false, $list->isArchived());
+        $this->assertSame($list, $list->setArchived(true));
+        $this->assertTrue(true);
```

Comment:
Required: here you are asserting that true is true, which is always going to pass. You should be asserting that the list is archived.